cubes/query/browser.py

```diff
@@ -75,6 +77,41 @@
 _ReportResult = Union[AggregationResult, Facts, JSONType, List[JSONType]] 
 
 
+class BrowserFeatureAction(Enum):
+    aggregate = 1
+    fact = 2
+    facts = 3
+    members = 4
+    cell = 5
+
+    @classmethod
+    def from_string(cls, actions: Iterable[str]) -> List['BrowserFeatureAction']:
+        if not actions:
+            return []
+
+        return [cls[action] for action in actions if action in cls.__members__.keys()]
```

Comment:
What about wrong actions? I think we should raise an error (internal/backend error as this has to be provided by backends).